configurations/kms-ear.yaml

```diff
@@ -1,11 +1,3 @@
-
-scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'KmsKeyProviderFactory', 'kms_host': 'kms_test'}"
-
-# config kms_hosts for AWS KMS service account
-append_scylla_yaml: |
-  kms_hosts:
-      kms_test:
-          master_key: 'alias/kms_encryption_test'
-          aws_region: 'us-east-1'
-
+server_encrypt: false
```

Comment:
JFYI: there is some problem with the `server_encrypt: true`
So, disabled it to unblock the testing of the feature added here.

Reply:
which problem is that ? the fact it crashes after reboot ? 
if so please take the lastest enterprise image to test with.

Reply:
Yes, the very first reboot was causing a crash.
Trying latest with the `server_encrypt: true`